### PR TITLE
CFE-1175: Add CI config for release-4.16 branch of ansible-operator-plugins repo

### PR DESCRIPTION
**Pull request by arkadeepsen**
This PR adds CI config for release-4.16 branch of ansible-operator-plugins repo.

### CODE REVIEW

File: ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-release-4.16.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+base_images:
+  ocp_4.17_base-rhel9:
```
Comment (chiragkyal) [inline ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-release-4.16.yaml:2]: Why not use 4.16 here as well?

Comment (arkadeepsen) [reply ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-release-4.16.yaml:2]: Ah, missed that one. Thanks for pointing it out.

Comment (arkadeepsen) [reply ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-release-4.16.yaml]: Updated it to 4.16.

Comment (chiragkyal) [reply ci-operator/config/openshift/ansible-operator-plugins/openshift-ansible-operator-plugins-release-4.16.yaml]: Thank you!

**Pull request merged**